packages/eslint/src/typescriptConfig.mjs

```diff
@@ -5,22 +5,6 @@ export const typescriptConfig = [
     {
         rules: {
             // Additional rules
-            '@typescript-eslint/no-unused-vars': [
-                'error',
-                {
-                    vars: 'all',
-                    args: 'none',
-                    ignoreRestSiblings: true,
-                    varsIgnorePattern: '^_',
-                },
-            ],
-            '@typescript-eslint/no-shadow': [
```

Comment:
It was not off, it was on only for new packages from moment I added it. Please bring it back at least for suite-native